src/components/services/HomeAssistant.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle">
+          {{ item.subtitle }}
+        </template>
+        <template v-else>
+          {{ details }}
+        </template>
+      </p>
+    </template>
+    <template #indicator>
+      <div v-if="status" class="status" :class="status">
+        {{ status }}
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "HomeAssistant",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    status: "",
+    msg: "",
+    version: "",
+    entities: 0,
+  }),
+  computed: {
+    headers: function () {
+      return {
+        'Authorization': `Bearer ${this.item.apikey}`,
+        'Content-Type': 'application/json',
+      };
+    },
+    details: function () {
+      const details = [];
+      const items = this.item.items;
+      const separator = this.item.separator;
+
+      for (const i in items) {
+        const key = items[i];
+
+        switch (key) {
+          case "version":
+            details.push(
+              `v${this.version}`,
+            );
+            break;
+          case "name":
+            details.push(
+              `${this.location_name}`,
+            );
+            break;
+          case "entities":
+            details.push(
+              `Entities: ${this.entities}`,
+            );
+            break;
+          default:
+            details.push(
+              `undefined key ${key} `,
+            );
+        }
+      }
+      
+      return details.join(separator);
+    },
+  },
+  created() {
+    this.fetchServerStatus().then(() => {
+      if (!this.item.subtitle && this.status !== "dead") {
+        if (!this.item.items) this.item.items = ["name", "version"];
+        if (!this.item.separator) this.item.separator = " ";
+        if (!this.item.logo) this.item.logo = `${this.item.url}/static/icons/favicon-192x192.png`;
```

Comment:
ESlint is not happy with the prop mutation (cf: https://github.com/bastienwirtz/homer/actions/runs/9179879689/job/25243741529?pr=783#step:6:12), I forgot about that in my first review, sorry. Could you see to use local variable or data variables for example? thanks!

Reply:
Ah sad, I really liked the idea of automatically grabbing the icon of home assistant if no logo was provided. I will fix it, thank you for your patience.

Reply:
Yeah that was a good idea! (could be applied to other service too. Thinking about it, in this case it should be harmless, we could disable the lint rules :thinking: 
I'll look into it, we can merge it like that for now, and change it back in the future if we are ok with the prop mutation.